widget/embedded/src/components/SwapDetailsAlerts/SwapDetailsAlerts.Warning.tsx

```diff
@@ -14,6 +14,7 @@ export function WarningAlert(props: WaningAlertsProps) {
         title={message.shortMessage}
         action={
           <Button
+            id="widget-swap-details-warning-alert-change-btn"
```

Comment:
I believe "change-network" is a much more appropriate name for the action.

Reply:
fixed